prime-router/metadata/fhir_mapping/hl7/datatypes/NM/Quantity-LengthOfStay.yml

```diff
@@ -0,0 +1,16 @@
+value:
+  type: STRING
+  valueOf: NM.1
+  expressionType: HL7Spec
+
+unit:
```

Comment:
There are other spots with this same problem. The POSITIVEINT doesn't really do anything here but I used that type as a mark for later. There's not a way to do math in the condition's currently. But since we forked the library we should be able to add that functionality. I'll make a ticket for it.

Reply:
Can you please link the ticket in this thread when created?

Reply:
I was just writing the ticket for this and took a look at how we might implement and found out the functionality does in fact exist! Only string comparisons are documented in the library docs but there are also int comparisons (see`io.github.linuxforhealth.core.expression.condition.ConditionBiPredicates`). So I'm going to update the mappings here to use `GREATER_THAN`

Reply:
Ran into a snag here. If I set a condition with `GREATER_THAN 0` then the value/unit/system/code fields are not populated which is what we want. I added an extension to keep the original value in the event that a non-positive value is sent. The problem is `value` is a required field so if it's blank then nothing else in the datatype will be created including the extension. With that in mind, I think we should keep the mapping how it is in this PR and leave the condition to future validation work. Thoughts?

Reply:
I agree, the mapping says we should "raise an error" in those cases, and that's not something we have any precedent for doing during the mapping, so it should be handled in validation. If we end up with negative values causing issues on receivers we can maybe handle that with receiver transforms down the line.